src/eval-frame/components/reps/react-table-styles.css

```diff
@@ -0,0 +1,46 @@
+.ReactTable {
```

Comment:
I am noticing that the `ReactTable` does not by default stretch to fill its space. Maybe this is a weird interaction w/ the grid?

![Screen Shot 2019-04-10 at 1 59 48 PM](https://user-images.githubusercontent.com/95735/55918041-79f04900-5ba6-11e9-9b7c-5a11c7813a93.png)

Reply:
i was playing around with this, and i believe that it's actually more readable to not stretch the table if there are just a couple columns. let's try dogfooding it a bit as-is -- i'm not committed to the way it is now, and if we think it will be better if it always stretches, we can make that change